frontend/src/components/pages/Scheduling/SelectDateTime.tsx

```diff
@@ -321,7 +415,7 @@ const SelectDateTime = ({
       )}
 
       {isDesktop ? (
-        <FormControl isRequired mb="2em">
+        <FormControl isRequired isInvalid={!!formErrors.frequency} mb="2em">
           <FormLabel fontWeight="600">
             How often will this donation occur?
           </FormLabel>
```

Comment:
In select can we add `isDisabled={isBeingEdited}` cuz user should not be able to change frequency when editing at least for phase 1? I am not sure how hard is this to do for mobile version